src/content/administration/dashboard.mdx

```diff
@@ -61,16 +61,16 @@ You can administer all namespaces in your Okteto instance from the Admin Dashboa
 For each namespace, an administrator will be able to:
 
 - View the name, owner, resources currently consumed, resources presently requested, and the last time it was accessed
-- Mark the namespace as `persistent` to prevent it from being automatically scaled to zero or deleted by the [garbage collector](/docs/administration/cleanup/#delete-unused-namespaces)
+- Mark the namespace as `persistent` to prevent it from being automatically scaled to zero or deleted by the [garbage collector](administration/cleanup.mdx#delete-unused-namespaces)
 - Get `read-only` access to any namespace managed by Okteto
 
 ## Secrets Section
 
-The Secrets section of the Admin dashboard lets you define cluster-wide secrets. Using the `$SECRET_NAME` syntax, you can refer to these secrets directly in your Okteto manifest. They are also injected into the [environment running](/docs/cloud/okteto-cli/#built-in-tools-when-deploying-to-okteto-cloud) the commands in the `deploy` section of the manifest.
+The Secrets section of the Admin dashboard lets you define cluster-wide secrets. Using the `$SECRET_NAME` syntax, you can refer to these secrets directly in your Okteto manifest. They are also injected into the [environment running](cloud/okteto-cli.mdx#built-in-tools-when-deploying-to-okteto-cloud) the commands in the `deploy` section of the manifest.
 
-> Secrets [defined by the user](/docs/cloud/secrets/) take precedence over the secrets defined here in case of a name collision.
+> Secrets [defined by the user](cloud/secrets.mdx) take precedence over the secrets defined here in case of a name collision.
```

Comment:
Asking to understand for future edits: 
In general we should 
- add the `.mdx` extension
- use path without `/docs/`

Reply:
We should point to the mdx file versus the URL. Docusaurus will take care of transforming `cloud/secrets.mdx` to the right URL.

Here’s the official doc: https://docusaurus.io/docs/versioning#link-docs-by-file-paths

Reply:
@huguestennier can we point to mdx + an anchor inside the page?

Reply:
Validated that anchors work by clicking on https://github.com/okteto/docs/pull/295/files#diff-cbd2fceab19489448408a5c30e5292108a85b65d5401fe0c5f2213bd8a54a59cR59